content/build/ci/github-actions/build-summary.md

```diff
@@ -0,0 +1,77 @@
+---
+title: GitHub Actions build summary
+descripion: Get an overview of your Docker Builds with GitHub Actions
+keywords: github actions, gha, build, summary, annotation
+---
+
+Docker's GitHub Actions for building and pushing images generate a job summary
+for your build that outlines the execution and materials used:
+
+- A summary showing the Dockerfile used, the build duration, and cache utilization
+- Inputs for the build, such as build arguments, tags, labels, and build contexts
+- For builds with [Bake](../../bake/_index.md), the full bake definition for the build
+
+![A GitHub Actions build summary](../images/gha_build_summary.png)
+
+Job summaries for Docker builds appear automatically if you use the following
+versions of the [Build and push Docker images](https://github.com/marketplace/actions/build-and-push-docker-images)
+or [Docker Buildx Bake](https://github.com/marketplace/actions/docker-buildx-bake)
+GitHub Actions:
+
+- `docker/build-push-action@v5.4`
```

Comment:
We don't currently publish `v<major>.<minor>`. So I think we should   set `v5.4.0`. Same for bake.